onnxscript/rewriter/fuse_pad_into_conv_test.py

```diff
@@ -0,0 +1,332 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+import typing
+import unittest
+
+import numpy as np
+import onnx_ir as ir
+import parameterized
+from onnx_ir.passes.common import onnx_checker, shape_inference
+
+from onnxscript.rewriter import pattern as orp
+from onnxscript.rewriter import testing
+from onnxscript.rewriter.fuse_pad_into_conv import (
+    fuse_pad_into_conv,
+    fuse_pad_into_conv_rule_set,
+    normalize_pad_format_conv,
+)
+
+
+def _clone_model(model: ir.Model) -> ir.Model:
+    return ir.from_proto(ir.to_proto(model))
+
+
+class FusePadConvBaseTest(unittest.TestCase):
+    @property
+    def rng(self):
+        return np.random.default_rng(20250522)
+
+    def get_conv_weights(self, shape: typing.Sequence[int], tape: ir.tape.Tape = None):
+        w = ir.tensor(self.rng.uniform(-0.5, 0.5, shape).astype("float32"), name="W")
+        if tape is not None:
+            w = tape.initializer(w)
+        return w
+
+    def build_model(
+        self,
+        op_type: str,
+        input_shape: ir.Shape,
+        weight_shape: typing.Sequence[int],
```

Comment:
```suggestion
        weight_shape: Sequence[int],
```

prefer importing the classes from the typing module: https://google.github.io/styleguide/pyguide.html#2241-exemptions